main.go

```diff
@@ -118,11 +132,25 @@ func run(ctx context.Context) error {
 				fmt.Fprintf(os.Stderr, "warning: could not load defaults from %q: %v\n", configPath, err)
 			}
 		}
-		if err := opt.LoadConfiguration(configBytes); err != nil {
-			fmt.Fprintf(os.Stderr, "warning: error loading configuration from %q: %v\n", configPath, err)
+		if len(configBytes) > 0 {
```

Comment:
does this mean we have a bug currently for users who don't have config file (like me) wrt to defaults ?

Reply:
We'd still run `opt.InitDefaults()` when there's no config files

Reply:
So, `opt.InitDefaults()` is called before this loading step and I was wondering if the content read is nil, will it reset the passed object. I tested, it doesn't, so no bugs.